tests/phpunit/tests/multisite/cleanDirsizeCache.php

```diff
@@ -205,13 +205,18 @@ function test_get_dirsize_cache_in_recurse_dirsize_upload() {
 			$this->assertSame( $size, $calc_size );
 
 			// `dirsize_cache` should now be filled after upload and recurse_dirsize() call.
-			$cache_path = untrailingslashit( str_replace( ABSPATH, '', $upload_dir['path'] ) );
+			$cache_path = untrailingslashit( str_replace( WP_CONTENT_DIR, '', $upload_dir['path'] ) );
 			$this->assertSame( true, is_array( get_transient( 'dirsize_cache' ) ) );
 			$this->assertSame( $size, get_transient( 'dirsize_cache' )[ $cache_path ] );
 
 			// Cleanup.
 			$this->remove_added_uploads();
 			restore_current_blog();
+
+			// aside: why do they repeat the same factory logic?
```

Comment:
`tearDown()` removes new objects in the `wp_posts` table when the transaction resets but doesn't delete the attachment files. It could be in the `tearDown()` I guess but maybe the contributor didn't want to run it each time as some tests don't create files.

Reply:
Just copied it over from the other multisite / upload related tests where it's mentioned that files will not be cleaned up automatically :-)

Reply:
👍🏻